templates/web/panels/image.tt.html

```diff
@@ -11,7 +11,10 @@
     <meta itemprop="imgid" content="[% image.type %]">
     
     <figcaption>
-        <a href="/cgi/product_image.pl?code=[% code %]&amp;id=[% image.id %]" title="Photo detail and attribution information">
+        <a href="https://nutripatrol.openfoodfacts.org/flag/image/?barcode=[% code %]&image_id=[% imgid %]&source=web&flavor=[% flavor %]" title="[% lang("action_report_image_to_nutripatrol") %]" style="text-decoration: none;">
```

Comment:
The nutripatrol URL must not be fixed. You have a NUTRIPATROL_URL settings for that (pass it in the template data)